# INSTALL_LuMP_Pulsar_SOFA_P0.patch
# patch file 0 for SOFA
# 2014 Oct 02  James Anderson  --- GFZ start
#
# Copyright (c) 2014 James M. Anderson <anderson@gfz-potsdam.de>
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
#
# Note that the SOFA software in this patch is presumably covered by a different
# copyright.  See http://www.iausofa.org/ for more information about 
# SOFA copyright issues.
#
################################################################################
diff -Naur sofa/20131202_c/f77/src/makefile sofa_new/20131202_c/f77/src/makefile
--- sofa/20131202_c/f77/src/makefile	2014-09-08 15:36:16.000000000 +0200
+++ sofa_new/20131202_c/f77/src/makefile	2014-10-02 05:44:24.665336849 +0200
@@ -52,7 +52,7 @@
 
 # Specify the installation home directory.
 
-INSTALL_DIR = $(HOME)
+INSTALL_DIR = $(MYDIR)
 
 # Specify the installation directory for the library.
 
@@ -63,7 +63,7 @@
 # (FF for subprograms and FX for executables) here.
 
 FC = gfortran
-FX = -O -Wall
+FX = $(MY_EXTRA_FFLAGS) -Wall -fPIC
 FF = -c $(FX)
 
 #----YOU SHOULDN'T HAVE TO MODIFY ANYTHING BELOW THIS LINE---------
@@ -799,4 +799,4 @@
 iau_ZR.o : zr.for
 	$(FC) $(FF) -o $@ zr.for
 
-#-----------------------------------------------------------------------
\ No newline at end of file
+#-----------------------------------------------------------------------
